src/nvim/ex_cmds.c

```diff
@@ -348,6 +348,7 @@ void ex_sort(exarg_T *eap)
   long deleted;
   colnr_T start_col;
   colnr_T end_col;
+  int sort_bin;                         /* sort on bin number */
```

Comment:
// comment, same for next comments (remember that linter requires changing adjacent lines). Also comment should not be that far to the right.